modules/installation-restricted-network-samples.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc
+
+[id="installation-restricted-network-samples_{context}"]
+= Using sample imagestreams in a restricted network installation
+
+Most imagestreams in the OpenShift namespace managed by the Samples Operator
+point to images located in the Red Hat registry at registry.redhat.io. Mirroring
+will not apply to these imagestreams.
+
+The `jenkins`, `jenkins-agent-maven`, and `jenkins-agent-nodejs` imagestreams do
+come from the install payload and are managed by the Samples Operator, so no
+further mirroring procedures are needed for those imagestreams.
+
+[NOTE]
+====
+The `cli`, `installer`, `must-gather`, and `tests` imagestreams, while
+part of the install payload, are not managed by the Samples Operator. These are
+not addressed in this procedure. 
+====
+
+.Prerequisites
+* Cluster administrator privileges
+
+.Procedure
+
+. Mirror images from registry.redhat.io associated with any imagestreams you need
+in the restricted network environment into one of the defined CRI-O mirrors.
+
+. Add the required trusted CAs for the CRI-O mirror in the cluster’s image
+configuration object.
+
+//(meaning https://github.com/openshift/api/blob/master/config/v1/types_image.go#L46 … add ref to associated doc)
+
+. Update the `samplesRegistry` field in the Samples Operator configuration object
```

Comment:
Need keep paragraph indent different with last level paragraph

Reply:
It's building correctly now, thanks!